indico/modules/rb/controllers/backend/misc.py

```diff
@@ -27,15 +29,25 @@
 from indico.modules.rb.util import build_rooms_spritesheet
 from indico.util.caching import memoize_redis
 from indico.util.i18n import get_all_locales
+from indico.util.string import sanitize_html
 from indico.web.flask.util import send_file, url_for
 
 
 class RHConfig(RHRoomBookingBase):
     def _process(self):
+        tos_url = None
+        url = legal_settings.get('tos_url')
+        if url:
+            tos_url = url
```

Comment:
this whole thing can be replaced with `tos_url = legal_settings.get('tos_url')` - and if the setting can be an empty string while you need `None`, just add `or None`